modin/pandas/test/test_series.py

```diff
@@ -770,7 +770,14 @@ def test_align(data):
 )
 def test_all(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.all(skipna=skipna), pandas_series.all(skipna=skipna))
+    # We currently do not accept None for skipna, while pandas warns users that it will
```

Comment:
Should we warn users as well in the case skipna=None?

Reply:
Do you mean we should allow `None` for now, and just do a `FutureWarning` like pandas does? I originally considered that, but then we'd have to change our code again when pandas officially deprecates it. Happy to make the change though if you think that's better!

Reply:
Yes, I think we should do a `FutureWarning` like pandas does.

Reply:
Sounds like a plan! I'll revert + update the code!